css-syntax/styles.css

```diff
@@ -0,0 +1,28 @@
+h1 {
```

Comment:
please add the `body` selector and provide the properties based on the instructions in the assignment (`color` and `font-family`)